README.md

```diff
@@ -1,142 +1,58 @@
+own custom components. The team maintains several npm packages.
+
+| Package                   | Description                                                                        | Docs             |
+|---------------------------|------------------------------------------------------------------------------------|------------------|
+| `@angular/cdk`            | Library that helps you author custom UI components with common interation patterns | [Docs][cdk-docs] |
+| `@angular/material`       | [Material Design][] UI components for Angular applications                         | [Docs][mat-docs] |
+| `@angular/google-maps`    | Angular components built on top of the [Google Maps JavaScript API][]              | [Docs][map-docs] |
+| `@angular/youtube-player` | Angular component built on top of the [YouTube Player API][]                       | [Docs][ytp-docs] |
+
 
 #### Quick links
-[Documentation, demos, and guides][aio] |
-[Google group](https://groups.google.com/forum/#!forum/angular-material2) |
+[Documentation, demos, and guides][mat-docs] |
+[Frequently Asked Questions][./FAQ] |
+[Community Google group](https://groups.google.com/forum/#!forum/angular-material2) |
 [Contributing](https://github.com/angular/components/blob/master/CONTRIBUTING.md) |
 [StackBlitz Template](https://stackblitz.com/fork/components-issue)
 
-### Getting started
+## Getting started
 
-See our [Getting Started Guide][getting-started]
-if you're building your first project with Angular Material.
+See our [Getting Started Guide][getting-started] if you're building your first project with Angular
+Material.
 
-Check out our [directory of design documents](https://github.com/angular/components/wiki/Design-doc-directory)
-for more insight into our process.
 
-If you'd like to contribute, you must follow our [contributing guidelines](https://github.com/angular/components/blob/master/CONTRIBUTING.md).
-You can look through the GitHub issues (which should be up-to-date on who is working on which features
-and which pieces are blocked) and make a comment.
+## Contributing
 
-Please see our [`help wanted`](https://github.com/angular/components/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
-label for a list of issues where we could use help from the community.
+If you'd like to contribute, please follow our [contributing guidelines][contributing]. Please see
+our [`help wanted`][help-wanted] label for a list of issues with good opportunities for
+contribution.
 
-#### High level stuff planned for Q4 2019 (Oct  - Nov):
+## High level work planned for Q4 2019 (Oct  - Dec):
 * Remove dependency on HammerJS
 * Finish remaining test harnesses for Angular Material components
 * Continuing to create new, API-compatible versions of the Angular Material components backed by
-MDC Web ([see @jelbourn's ng-conf talk](https://youtu.be/4EXQKP-Sihw?t=891)).
+[MDC Web][] ([see @jelbourn's ng-conf talk](https://youtu.be/4EXQKP-Sihw?t=891)).
 * New `@angular/google-maps` package
```

Comment:
I think this needs to be removed, right? It shipped!

Reply:
It was still part of this quarter, so I'm going to leave it until January